app/views/layouts/application.html.erb

```diff
@@ -28,7 +28,7 @@
         <% end %>
       </div>
       <nav class="header--menu">
-        <%= link_to 'Questions', questions_path, class: 'header--item' %>
+        <%= link_to 'Categories', categories_path, class: 'header--item' %>
         <%= link_to 'Users', users_path, class: 'header--item' %>
         <%= link_to 'Search', search_path, class: 'header--item' %>
 
```

Comment:
There is still a Meta link in the header. Is that intentional?

Reply:
The link has been fixed, but the header should be split into a .header (logo, Categories, Users, Search | Help, Flags, User tool) and a .category-header (Categories as tabs and each with: Posts, Tags | New Post)